src/OpenTelemetry.Api.ProviderBuilderExtensions/Trace/OpenTelemetryDependencyInjectionTracingServiceCollectionExtensions.cs

```diff
@@ -26,17 +26,15 @@ public static class OpenTelemetryDependencyInjectionTracingServiceCollectionExte
 {
     /// <summary>
     /// Registers an action used to configure the OpenTelemetry <see
-    /// cref="TracerProviderBuilder"/> used to create the <see
-    /// cref="TracerProvider"/> for the <see cref="IServiceCollection"/> being
-    /// configured.
+    /// cref="TracerProviderBuilder"/>.
```

Comment:
Good improvement! Was confusing before "used to configure ... used to create the ... for the ... being configured" 😆